src/Microsoft.AspNet.Diagnostics.Elm/project.json

```diff
@@ -17,5 +16,6 @@
                 "System.Threading": "4.0.11-beta-*"
             }
         }
-    }
+    },
+    "resource": ["Views/*.cshtml"]
```

Comment:
If left in it seems the build process tries to treat them as real cshtml files, and since they're not this results in errors like the one [https://travis-ci.org/aspnet/Diagnostics/builds/85435845](here). I'm good moving to compileExclude, but might it be even better to just change the extension to not be cshtml since they're really not?